### PR TITLE
fix: remove two way binding (DEV-4339

### PR DESCRIPTION
**Pull request by domsteinbach**
resolves DEV-4339

### CODE REVIEW

**Review by irmastnt — APPROVED** [review]

**Review by derschnee68 — APPROVED** [review]

File: libs/vre/shared/app-resource-page/src/lib/resource.component.ts [hunk 1]
```diff
@@ -39,10 +37,11 @@ export class ResourceComponent implements OnInit {
   @Input({ required: true }) resource!: DspResource;
   @Input({ required: true }) isDifferentResource!: boolean;
   representationsToDisplay!: ReadFileValue;
-  isCompoundNavigation: boolean | null = null;
+  isCompoundNavigation: boolean = false;
```
Comment (derschnee68) [inline libs/vre/shared/app-resource-page/src/lib/resource.component.ts:40]: isCompoundNavigation should be undefined (or null) by default

File: libs/vre/shared/app-resource-page/src/lib/compound/compound-viewer.component.ts [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
+import { Component, OnInit } from '@angular/core';
+import { DspResource } from '@dasch-swiss/vre/shared/app-common';
 import { CompoundService } from './compound.service';
 
 @Component({
   selector: 'app-compound-viewer',
   template: `
     <ng-container *ngIf="compoundService.compoundPosition">
-      <ng-container *ngIf="compoundService.incomingResource as incomingResource">
+      <ng-container *ngIf="incomingResource">
```
Comment (derschnee68) [inline libs/vre/shared/app-resource-page/src/lib/compound/compound-viewer.component.ts:9]: <ng-container *ngIf="compoundService.incomingResource$ | async as incomingResource">

File: libs/vre/shared/app-resource-page/src/lib/compound/compound.service.ts [hunk 3]
```diff
@@ -5,14 +5,17 @@ import { IncomingService } from '@dasch-swiss/vre/shared/app-common-to-move';
...
 
 /**
  * Service to handle compound resources, which are resources that are composed of multiple resources.
  */
 @Injectable()
 export class CompoundService {
   compoundPosition?: DspCompoundPosition;
-  incomingResource?: DspResource;
+
+  private _incomingResource = new BehaviorSubject<DspResource | undefined>(undefined);
```
Comment (derschnee68) [inline libs/vre/shared/app-resource-page/src/lib/compound/compound.service.ts:17]: Could be a Subject, but it's ok otherwise

**Review by derschnee68 — APPROVED** [review]

File: libs/vre/shared/app-resource-page/src/lib/resource.component.ts (hunk 1 above)
Comment (domsteinbach) [reply libs/vre/shared/app-resource-page/src/lib/resource.component.ts:40]: It does not need to. The components display logic should be changed instead imho.

**Pull request closed without merging**